docs/changelog.txt

```diff
@@ -94,6 +94,9 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 - ``argparse.processArgsGetopt()``: you can now have long form parameters that are not an alias for a short form parameter. For example, you can now have a parameter like ``--longparam`` without needing to have an equivalent one-letter ``-l`` param.
 - ``dwarfmode.enterSidebarMode()``: ``df.ui_sidebar_mode.DesignateMine`` is now a suported target sidebar mode
 
+## Documentation
```

Comment:
Make sure you add this entry to the "Future" section above. Right now it's in the release notes for the last release (0.47.05-r4)